### ISSUE TITLE
CONTROLLER LAG

### ISSUE THREAD

**Issue by Dat904Kid** [Jays2Kings/DS4Windows#63]
I play a fighting game and when I try to do a combo like Square, Square, Square. Instead of hitting the combo, I does a hold attack as if I held the Square button for 2 seconds. Please help and fix.

**Comment by mirh** [comment]
Possibly related to #62

**Comment by Dat904Kid** [comment]
Nope..I don't have any latency issues or disconnecting and connecting issues. I've deleted it and re-downloaded it multiple times. I'm jus getting a delay wen I'm trying to do a simple fast combo like Square, Square, Square or Triangle, Triangle. It's doing a hold attack instead of the combos I just listed.

**Comment by RCal-Dev** [comment]
Are you connected via Bluetooth or wired?

**Comment by Dat904Kid** [comment]
Im wired but whenever I use the Bluetooth Dongle it's even worse. I would have latency issues and the controller would lag so hard that it would be connected but it would stop working for like 4 to 5 seconds and I would get my ass beat in the game.

**Comment by RCal-Dev** [comment]
Have you tried reinstalling the SCP driver?  Maybe you have many programs running in the background taking resources?

**Comment by Dat904Kid** [comment]
Yea I have done that as well. Uninstalled the SCP and reinstalled it, then restarted my computer. I just think the creators of DS4 needs to update it to solve our issues.

**Comment by cjrobe** [comment]
SOLVED. At least for me. By default, Windows puts computers, even desktop computers, on "Balanced Power Mode." I put it on high performance and my Input Delay went from 30-80ms to 2ms.

@Dat904Kid

**Comment by Dat904Kid** [comment]
How u put it to high performance?

**Comment by cjrobe** [comment]
Search control panel for "power settings" and click it. Then change it to high performance mode.

Unfortunately after a reboot, I'm back to getting 30-80ms Input Delay :(. I don't understand why it's so inconsistent.
